node/node.go

```diff
@@ -802,6 +794,21 @@ func (n *Node) apis() []rpc.API {
 			Public:    true,
 		},
 	}
+	debugRpcApi := []rpc.API{
+		{
+			Namespace: "unsafedebug",
+			Version:   "1.0",
+			Service:   debug.Handler,
+		}, {
+			Namespace: "unsafedebug",
```

Comment:
Not "debug"?

Reply:
Right, renamed [f34a3f3](https://github.com/klaytn/klaytn/pull/1746/commits/f34a3f3117c0119cb1538ca0c26b3e1fa18dc9d2)